plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb-v4.metrics.test.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// for testing locally "npm run docker:start"
+
+import { MongoDBInstrumentation } from '../src';
+
+// TODO: use test-utils after the new package has released.
+import {
+  AggregationTemporality,
+  DataPointType,
+  InMemoryMetricExporter,
+  MeterProvider,
+  PeriodicExportingMetricReader,
+  ResourceMetrics,
+} from '@opentelemetry/sdk-metrics';
+
+import * as mongodb from 'mongodb';
+
+const otelTestingMeterProvider = new MeterProvider();
+const inMemoryMetricsExporter = new InMemoryMetricExporter(
+  AggregationTemporality.CUMULATIVE
+);
+const metricReader = new PeriodicExportingMetricReader({
```

Comment:
Nit: IIUC, a pulling metric reader might be better suited for testing usage since it can pull metrics proactively instead of awaiting timeouts.